src/util/crypto-node.ts

```diff
@@ -58,3 +66,15 @@ export function generateSharedSecretFromToken (seed: Buffer, token: Buffer): Buf
   const sharedSecret = hmacSync(keygen, token)
   return sharedSecret
 }
+
+export function generateReceiptHMAC (secret: Buffer, receipt: Buffer): Buffer {
```

Comment:
calling this buffer a receipt is a little confusing because a full receipt contains the hmac

Reply:
renamed to `message`
https://github.com/interledgerjs/ilp-protocol-stream/pull/133/commits/7856838aa567b26812c49e7c41484f4c092ae9bf